config/crd/bases/multicluster.odf.openshift.io_mirrorpeers.yaml

```diff
@@ -68,7 +68,9 @@ spec:
                 type: boolean
               schedulingIntervals:
                 description: SchedulingIntervals is a list of intervals at which mirroring
-                  snapshots are taken
+                  snapshots are taken. SchedulingIntervals has been deprecated and
+                  now only resides in DRPolicy. Any changes to this field will not
+                  affect the cluster state
```

Comment:
These changes should be part of the other commit that only has generated changes.